sql/core/src/test/resources/sql-tests/inputs/subquery/negative-cases/subq-input-typecheck.sql

```diff
@@ -0,0 +1,47 @@
+-- The test file contains negative test cases
+-- of invalid queries where error messages are expected.
```

Comment:
These negative test cases issue the exactly same errors? I mean, before this PR and after this PR.